Plugins/Json/include/Acts/Plugins/Json/GeometryHierarchyMapJsonConverter.hpp

```diff
@@ -52,7 +52,7 @@ class GeometryHierarchyMapJsonConverter {
   ///
   /// @param container Geometry hierarchy map that should be encoded
   /// @return Encoded Json object
-  nlohmann::json toJson(const Container& container) const;
+  nlohmann::ordered_json toJson(const Container& container) const;
```

Comment:
Is the use of `ordered_json` required for all objects in a hierarchy or only for the ones that should be ordered? If it is the latter, this can stay just as `json` since only each item needs to be ordered.

Reply:
I think if you do `json = ordered_json` you lose the ordering, so the final object you return has to be of type `ordered_json`. To be honest I haven't really found documentation on `ordered_json` but I just tested it and changing back to `json` doesn't not work in our case.

Reply:
Ok, thanks for checking.